### ISSUE TITLE
Update all_search to use the focus indicator from Lux design system

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/allsearch_frontend#323]
### Steps to recreate
Visit any search result page, i.e. https://allsearch.princeton.edu/?q=dogs. 

### Severity
Medium per John Jameson. 

### Issues
:focus indicator does not have enough contrast to be visible for some users.

<img width="618" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0dZQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4da3f9aae5e9c448d630cadcf0f650104204bd07/Screenshot%202024-06-27%20at%208.40.52%E2%80%AFAM.png" alt="Screenshot 2024-06-27 at 8.40.52 AM.png" />

### Severity
Critical/blocking per John Jameson. 

### WCAG Conformance level (A, AA, AAA)
Visual distinctions used to identify elements and states (e.g., a link's underline, its hover color and focus outline) each have a contrast ratio of at least 3:1 against their background and alternate states. (1.4.11 AA) 

### User impact
Low vision users are challenged to use the site. 

### Recommended fix
Use a darker color for the focus ring.

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

**Comment by christinach** [comment]
closed by https://github.com/pulibrary/allsearch_frontend/pull/347
